stories/assets/js/sticky.js

```diff
@@ -1,20 +1,26 @@
-function sticky_relocate() {
-    var window_top = $(window).scrollTop();
-    var breadcrumb = $('.breadcrumb__wrapper');
-    if (breadcrumb.length) {
-        var div_top = $('.breadcrumb__wrapper').offset().top;
-        if (window_top > div_top) {
-            $('.sticky').addClass('stick');
-            if(window_top > 2000){
-                $('.sticky').removeClass('stick');
-            }
-        } else {
-            $('.sticky').removeClass('stick');
+$( document ).ready(function() {
+    var $sticky = $('.sticky');
+    var $stickyrStopper = $('.footer');
+    if (!!$sticky.offset()) {  
+      var generalSidebarHeight = $sticky.innerHeight();
+      var stickyTop = $sticky.offset().top;
+      var stickOffset = 0;
+      var stickyStopperPosition = $stickyrStopper.offset().top;
+      var stopPoint = stickyStopperPosition -(generalSidebarHeight +stickOffset);
+      var diff = stopPoint -($('footer').innerHeight()-150);
+      
+      $(window).scroll(function(){
+        var windowTop = $(window).scrollTop();
+        if($(window).width() > 834){
```

Comment:
@neelesh-tippe kindly refrain from using hardcoded values. Kindly use a global config file to define viewport values.